guides/common/modules/proc_migrating-project-or-smartproxy-using-clone.adoc

```diff
@@ -0,0 +1,22 @@
+[id="migrating-{project-context}-or-proxy-using-clone_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9 using clone
+
+You can clone your existing {ProjectServer} and {SmartProxyServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+Create a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
```

Comment:
This whole section describes that you _can_ use cloning to back up a Smart Proxy Server, doesn't it? So how does this note fit in?

Reply:
My bad. I think I edited and committed without saving the changes.